frontend/.gitignore

```diff
@@ -10,5 +10,19 @@
 next-env.d.ts
 
 # testing & coverage
+
+# jest coverage
 /coverage
 
+# playwright coverage
+/coverage-pw
+
+# playwright component testing coverage
+/coverage-ct
+
+node_modules/
```

Comment:
you shouldn't need this, I believe it's filtered at the root of the project
https://github.com/crt25/collimator/blob/main/.gitignore#L27